omnia/bin/oracle-transporter-ssb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Why no `libexec`?

Reply:
My thought was that we might as well expose the Transport implementation from the omnia package.

Reply:
Indeed we can